src/protocol/templates/ping1d.py.in

```diff
@@ -51,8 +51,8 @@ class Ping1D(object):
             b = self.iodev.read()
 
             if self.parser.parse_byte(ord(b)) == pingmessage.PingParser.NEW_MESSAGE:
-                self.handle_message(self.parser.rxMsg)
-                return self.parser.rxMsg
+                self.handle_message(self.parser.rx_msg)
```

Comment:
how did your linter not catch this?

Reply:
or does the linter not give hits for this and it was all manual?

Reply:
It does give hints, I don't know what happened in this case